app/create_app.py

```diff
@@ -114,6 +118,34 @@ def inject_global_constants():
         health = Healthcheck(flask_app)
         health.add_check(FlaskRunningChecker())
 
+        @flask_app.before_request
```

Comment:
Beautiful!

Reply:
Luckily someone made the before_request decorator for me...!